config/chime_science_run_gpu.yaml

```diff
@@ -3966,3 +3969,1029 @@ input_reorder:
   - [1949, 2045, FCC070901]
   - [1950, 2046, FCC070902]
   - [1951, 2047, FCC070903]
+
+feed_positions: 
+[[-33.04813181029521, -38.82107710831004],
+ [-33.04775410717911, -38.51627734233186],
+ [-33.04737640406301, -38.21147757635368],
+ [-33.04699870094691, -37.90667781037549],
+ [-33.04662099783081, -37.60187804439731],
+ [-33.04624329471471, -37.29707827841912],
+ [-33.045865591598606, -36.99227851244094],
+ [-33.045487888482505, -36.68747874646275],
+ [-33.045110185366404, -36.38267898048457],
+ [-33.04473248225031, -36.07787921450638],
+ [-33.04435477913421, -35.7730794485282],
+ [-33.04397707601811, -35.46827968255001],
+ [-33.04359937290201, -35.16347991657183],
+ [-33.04322166978591, -34.85868015059364],
+ [-33.04284396666981, -34.55388038461546],
+ [-33.042466263553706, -34.24908061863727],
+ [-33.042088560437605, -33.94428085265909],
+ [-33.041710857321505, -33.6394810866809],
+ [-33.041333154205404, -33.33468132070272],
+ [-33.0409554510893, -33.02988155472453],
+ [-33.0405777479732, -32.72508178874635],
+ [-33.0402000448571, -32.42028202276816],
+ [-33.039822341741, -32.11548225678998],
+ [-33.03944463862491, -31.810682490811796],
+ [-33.03906693550881, -31.505882724833615],
+ [-33.038689232392706, -31.20108295885543],
+ [-33.038311529276605, -30.896283192877245],
+ [-33.037933826160504, -30.59148342689906],
+ [-33.0375561230444, -30.286683660920875],
+ [-33.0371784199283, -29.98188389494269],
+ [-33.0368007168122, -29.677084128964506],
+ [-33.0364230136961, -29.37228436298632],
+ [0, 0],
```

Comment:
We might want to have a check in the `compute_phases` function to set feeds without a position to have [0,0] for there phase to null those values in the beamformer.   Since I'm not sure if we can always expect that feeds without positions are also nulled in the gains (although most of the time I'd expect that to be the case).

Reply:
Done!